atc/job_config.go

```diff
@@ -24,8 +24,9 @@ type JobConfig struct {
 }
 
 type BuildLogRetention struct {
-	Builds int `json:"builds,omitempty"`
-	Days   int `json:"days,omitempty"`
+	Builds                 int `yaml:"builds,omitempty" json:"builds,omitempty" mapstructure:"builds"`
+	MinimumSucceededBuilds int `yaml:"minimum_succeeded_builds,omitempty" json:"minimum_succeeded_builds,omitempty" mapstructure:"minimum_succeeded_builds"`
+	Days                   int `yaml:"days,omitempty" json:"days,omitempty" mapstructure:"days"`
```

Comment:
Is there a reason to add `yaml` and `mapstructure` here? I think in general we are trying to avoid using mapstructure.

Not sure about `yaml` but take `PlanConfig` for example it is also `json` only.

Reply:
@xtremerui Thanks for the comment. I actually don't know. I added only `MinimumSucceededBuilds`, so I followed the existing fields `Build` and `Days`. Looks like in the latest code, `mapstructure` had been removed from `Build` and `Days`, I'll update the PR accordingly.